notes.md

```diff
@@ -586,6 +586,9 @@ Encryption in Matroska is designed in a very generic style to allow people to
 implement whatever form of encryption is best for them. It is possible to use the
 encryption framework in Matroska as a type of DRM (Digital Rights Management).
 
+Matroska itself doesn't create its own encryption system. It uses other existing
+encryption systems, with their data wrapped inside Matroska Elements.
+
```

Comment:
```suggestion
This document does not specify any kind of standard for encrypting elements.
The issue of key scheduling, authorisation, and authentication are out of scope.
External entities have used these elements in proprietary ways.

```

Reply:
The issues [plural] of key scheduling etc. are out of scope.

Reply:
Applied and updated the author.